tests/test_settings.py

```diff
@@ -20,22 +20,31 @@
 
 from arches.settings import *
 
-try:
-    from django.utils.translation import gettext_lazy as _
-except ImportError:  # unable to import prior to installing requirements
-    pass
+from django.utils.translation import gettext_lazy as _
```

Comment:
From the git blame, the try/except was here for when we had setup.py, but we've migrated to pyproject.toml